Backpack/src/main/java/net/skyscanner/backpack/flare/BpkFlare.kt

```diff
@@ -176,13 +211,13 @@ open class BpkFlare @JvmOverloads constructor(
     val height = height.toFloat()
     val radiusHeight = radiusMask.height
     val radiusWidth = radiusMask.width
-    val radiusHalfHeight = (radiusHeight / 2).toFloat()
-    val radiusHalfWidth = (radiusHeight / 2).toFloat()
+    val radiusHalfHeight = radiusHeight.toFloat() / 2
```

Comment:
This fixes a little issue that was causing it to render a line on the the right borders. It's visible on the docs site. https://backpack.github.io/components/flare?platform=android#rounded